### PR TITLE
PR for Homepage

### PR DESCRIPTION
**Pull request by SK02K1**
# Homepage added

### Features

- Fetch featured categories from firebase firestore DB.
- Theme context for toggling app theme and saving theme value in local storage for future.
- Sound effect on theme toggle.
- Change document title on page mounting using useDocumentTItle custom hook.

Screenshot 2022-05-07 at 12 00 03 AM

Screenshot 2022-05-07 at 12 01 45 AM

### CODE REVIEW

File: src/components/CategoryCard/CategoryCard.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+.overlay {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: linear-gradient(0deg, #00000088 30%, #ffffff44 100%);
+  color: #fff;
```
Comment (Nikhil-Nirmal) [inline src/components/CategoryCard/CategoryCard.css]: use color variables

Comment (simran61) [inline src/components/CategoryCard/CategoryCard.css]: you can try using css variables instead of hexcodes

**Review by simran61 — APPROVED** [review]
Looks good
clean folder structure
easily readable code

**Review by Nikhil-Nirmal — APPROVED** [review]
Everything Else looks good

**Pull request merged**